page-changelog.php

```diff
@@ -181,6 +283,29 @@
 Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/10.0/owncloud">10.0 Release Channel</a> and <a href="https://download.owncloud.org/download/repositories/stable/owncloud/">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
 
 <a name="latest9.1"></a></li>
+      <h3>9.1.7 RC2</h3>
```

Comment:
9.1.7 is already in final, please remove RC2. I suggest putting the date of tomorrow as release date, assumiing that this will be announced by then